query_plan.go

```diff
@@ -98,19 +109,28 @@ func getAllMetadataString(node *Node) string {
 	return fmt.Sprintf(`(%s)`, strings.Join(fields, ", "))
 }
 
-func renderTree(tree treeprint.Tree, node *Node) {
+func renderTree(tree treeprint.Tree, linkType string, node *Node) {
 	if !node.IsVisible() {
```

Comment:
Instead of separating Link into `linkType` and `node`, how about passing link itself?
```
func renderTree(tree treeprint.Tree, link *Link)
```

Reply:
IMHO, separating seems more simpler because:
* Less indirection(or temporal variable)
* No needs of a root Link
Example:
https://github.com/cloudspannerecosystem/spanner-cli/commit/0ffcc6834022574f847e93e39ede039e9f32c3e1

Reply:
Makes sense. Thank you for trying it.